pkg/shortnames/shortnames_test.go

```diff
@@ -470,6 +490,12 @@ func TestResolveLocally(t *testing.T) {
 		SystemRegistriesConfDirPath: "testdata/this-does-not-exist",
 		UserShortNameAliasConfPath:  tmp.Name(),
 	}
+	sysResolveToDockerHub := &types.SystemContext{
```

Comment:
Note to self: I didn’t actually review the test cases, I expect them to change if `localhost/` is not used in the Docker case.